lib/src/database/indexeddb_box.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: deprecated_member_use
```

Comment:
Hmm, not sure if it wouldn't be better to directly migrate everything to web. In the SDK actually we don't need to rush for the dart upgrade. Maybe worth another issue I guess? @asabri97